### ISSUE TITLE
Crash Log

### ISSUE THREAD

**Issue by Pieftw247** [HearthSim/Hearthstone-Deck-Tracker#2543]
```
########## 5/30/2016 10:54:34 AM ##########
System.ComponentModel.Win32Exception (0x80004005): Access is denied
   at System.Diagnostics.ProcessManager.OpenProcess(Int32 processId, Int32 access, Boolean throwIfExited)
   at System.Diagnostics.Process.GetProcessHandle(Int32 access, Boolean throwIfExited)
   at System.Diagnostics.Process.OpenProcessHandle(Int32 access)
   at System.Diagnostics.Process.get_Handle()
   at HearthMirror.ProcessView..ctor(Process proc)
   at HearthMirror.Mirror.get_View()
   at HearthMirror.Reflection.TryGetInternal[T](Func`1 action, Boolean clearCache)
   at HearthMirror.Reflection.GetSelectedDeckInMenu()
   at Hearthstone_Deck_Tracker.LogReader.Handlers.FullScreenFxHandler.AutoSelectDeckById(Boolean import)
   at Hearthstone_Deck_Tracker.LogReader.Handlers.FullScreenFxHandler.Handle(LogLineItem logLine, IGame game)
   at Hearthstone_Deck_Tracker.LogReader.LogReaderManager.ProcessNewLines()
   at Hearthstone_Deck_Tracker.LogReader.LogReaderManager.<StartLogReaders>d__20.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
10:53:33 AM|Info|OverlayWindow.Update >> Refreshed overlay topmost status
10:53:34 AM|Info|MainWindow.SelectDeck >> Switched to deck: Amnesiac OTK Warrior[Starladder] (v1.0)
10:53:34 AM|Warning|LogReaderManager.Stop >> LogReaders could not be stopped, stop already in progress.
10:53:34 AM|Info|GameV2.Reset >> -------- Reset ---------
10:53:34 AM|Info|Player.Name >> [Player] 
10:53:34 AM|Info|Player.Name >> [Opponent] 
10:53:34 AM|Info|OpponentSecrets.ClearSecrets >> Cleared secrets
10:53:35 AM|Info|DataIssueResolver.RemoveDuplicateMatches >> Checking for duplicate matches...
10:53:35 AM|Info|BackupManager.Run >> Running BackupManager
10:53:35 AM|Info|BackupManager.Run >> Backup for today already exists
10:53:35 AM|Info|PluginManager.LoadPlugins >> Loading Plugins...
10:53:35 AM|Info|PluginWrapper.Load >> Loading Anyfin Can Happen Calculator
10:53:35 AM|Info|PluginWrapper.IsEnabled >> Enabled Anyfin Can Happen Calculator
10:53:35 AM|Info|PluginWrapper.Load >> Loading Hearth Arena Uploader
10:53:35 AM|Info|PluginWrapper.IsEnabled >> Enabled Hearth Arena Uploader
10:53:35 AM|Info|PluginWrapper.Load >> Loading Arena Helper
10:53:35 AM|Info|Plugin.LoadData >> Arena Helper: Using userdata version of tierlist
10:53:35 AM|Info|PluginWrapper.IsEnabled >> Enabled Arena Helper
10:53:35 AM|Info|PluginWrapper.Load >> Loading Stats Converter
10:53:35 AM|Info|PluginWrapper.IsEnabled >> Enabled Stats Converter
10:53:35 AM|Info|GitHub.CheckForUpdate >> HearthSim/Hearthstone-Deck-Tracker: Checking for updates (current=0.15.1.0, pre-release=False)
10:53:35 AM|Info|Plugin.CheckPluginUpdate >> https://github.com/rembound/Arena-Helper/releases/download/0.8.0/ArenaHelper.v0.8.0.zip
10:53:35 AM|Info|GitHub.CheckForUpdate >> HearthSim/Hearthstone-Deck-Tracker: We are up-to-date (latest=v0.15.1, pre-release=False)
10:53:38 AM|Info|BackupManager.Run >> Running BackupManager
10:53:38 AM|Info|BackupManager.Run >> Backup for today already exists
10:53:38 AM|Info|OverlayWindow.Update >> Refreshed overlay topmost status
10:53:38 AM|Info|OverlayWindow.Update >> Refreshed overlay topmost status
10:53:42 AM|Info|LogReader.ReadLogFile >> Found LoadingScreen.log.
10:53:43 AM|Info|GameV2.CurrentMode >> LOGIN
10:53:45 AM|Info|LogReader.ReadLogFile >> Found Rachelle.log.
10:53:53 AM|Info|LogReader.ReadLogFile >> Found FullScreenFX.log.
10:54:04 AM|Info|OverlayWindow.Update >> Refreshed overlay topmost status
10:54:06 AM|Info|GameV2.CurrentMode >> HUB
10:54:10 AM|Info|OverlayWindow.Update >> Refreshed overlay topmost status
10:54:12 AM|Info|GameV2.CurrentMode >> TOURNAMENT
10:54:12 AM|Error|DeckImporter.FromConstructed >> System.ComponentModel.Win32Exception (0x80004005): Access is denied
   at System.Diagnostics.ProcessManager.OpenProcess(Int32 processId, Int32 access, Boolean throwIfExited)
   at System.Diagnostics.Process.GetProcessHandle(Int32 access, Boolean throwIfExited)
   at System.Diagnostics.Process.OpenProcessHandle(Int32 access)
   at System.Diagnostics.Process.get_Handle()
   at HearthMirror.ProcessView..ctor(Process proc)
   at HearthMirror.Mirror.get_View()
   at HearthMirror.Reflection.TryGetInternal[T](Func`1 action, Boolean clearCache)
   at HearthMirror.Reflection.GetDecks()
   at Hearthstone_Deck_Tracker.Importing.DeckImporter.GetConstructedDecks()
   at Hearthstone_Deck_Tracker.Importing.DeckImporter.FromConstructed()
10:54:16 AM|Info|OverlayWindow.Update >> Refreshed overlay topmost status
```
I unaware of what triggers the crash. The crash seems to be randomly occurring.

**Comment by Epix37** [comment]
Please try if running HDT as administrator helps.

**Comment by Epix37** [comment]
Also, if you are running Battle.net/Hearthstone as administrator: try running that as a normal user.

**Comment by Epix37** [comment]
Fixed with v0.15.2
